src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/ReadsPreprocessingPipeline.java

```diff
@@ -66,6 +67,14 @@
     @Argument(doc = "the known variants", shortName = "BQSRKnownVariants", fullName = "baseRecalibrationKnownVariants", optional = true)
     protected List<String> baseRecalibrationKnownVariants;
 
+
+    @Argument(doc = "See MarkDuplicatesDataflow",
```

Comment:
These should be extracted to their own `ArgumentCollectionDefinition` so the documentation doesn't have to be copied around.